Dockerfile

```
@@ -65,6 +65,38 @@ COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 ENTRYPOINT ["./ec2-resource-agent"]
 
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Builds the EC2 karpenter resource agent image
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as ec2-karpenter-resource-agent
+
+RUN yum install -y git make tar 
```

If you do an update, you can keep the image size a little smaller if you add `&& yum -y clean all && rm -fr /var/cache` to the end here.

---

```suggestion
RUN yum install -y git-core make tar 
```
You can also save some space with the git core sub-package.